cli/user/README.md

```diff
@@ -94,7 +95,52 @@ All available commands:
 	stream read scoped-stream-name [group-similar] [timeout-in-seconds]: Reads all Events from a Stream and then tails the Stream.
 ```
 
+You can try out commands such as:
+```
+> scope create example
+Scope 'example' created successfully.
+```
+
 For more info on how the Pravega User CLI works, please visit [this page](https://github.com/pravega/pravega/wiki/Pravega-User-CLI).
 
+## Pravega CLI on Kubernetes
+The pravega-cli can be accessed through the kubernetes pod such as the segmentstore pod.
+
+
+Once in the pod, you can run the pravega-cli under admin mode
```

Comment:
This looks copy/pasted from the other README, as you are stating the same exact wording, which is not correct `pravega-cli under admin mode`.